### ISSUE TITLE
command taskExplorer.refresh not found

### ISSUE THREAD

**Issue by mfreeman-xtivia** [spmeesseman/vscode-taskexplorer#61]
Installed in WSL mode, opened taskExplorer view in a Gradle project, and received above error

**Comment by spmeesseman** [comment]
@mfreeman-xtivia can u please test in 1.22 and let me know any difference?

**Comment by tkadur** [comment]
I am also experiencing this error running v1.22.0 on WSL. My project uses a Makefile, not Gradle.

<details>
<summary>Log output</summary>

```
[2020-02-03 11:42:14.889] [exthost] [error] Activating extension spmeesseman.vscode-taskexplorer failed due to an error:
[2020-02-03 11:42:14.892] [exthost] [error] Error: Cannot find module 'original-fs'
Require stack:
- /home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/util.js
- /home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/taskTree.js
- /home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/extension.js
- /home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/loader.js
- /home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/bootstrap-amd.js
- /home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
	at Function.Module._load (internal/modules/cjs/loader.js:527:27)
	at Function.t._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:836:851)
	at Function.n._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:807:164)
	at Function.i._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:803:618)
	at Module.require (internal/modules/cjs/loader.js:681:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Object.<anonymous> (/home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/util.js:14:23)
	at Module._compile (internal/modules/cjs/loader.js:774:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
	at Module.load (internal/modules/cjs/loader.js:641:32)
	at Function.Module._load (internal/modules/cjs/loader.js:556:12)
	at Function.t._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:836:851)
	at Function.n._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:807:164)
	at Function.i._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:803:618)
	at Module.require (internal/modules/cjs/loader.js:681:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Object.<anonymous> (/home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/taskTree.js:17:14)
	at Module._compile (internal/modules/cjs/loader.js:774:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
	at Module.load (internal/modules/cjs/loader.js:641:32)
	at Function.Module._load (internal/modules/cjs/loader.js:556:12)
	at Function.t._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:836:851)
	at Function.n._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:807:164)
	at Function.i._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:803:618)
	at Module.require (internal/modules/cjs/loader.js:681:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Object.<anonymous> (/home/tkadur/.vscode-server/extensions/spmeesseman.vscode-taskexplorer-1.22.0/out/extension.js:17:20)
	at Module._compile (internal/modules/cjs/loader.js:774:30)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
	at Module.load (internal/modules/cjs/loader.js:641:32)
	at Function.Module._load (internal/modules/cjs/loader.js:556:12)
	at Function.t._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:836:851)
	at Function.n._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:807:164)
	at Function.i._load (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:803:618)
	at Module.require (internal/modules/cjs/loader.js:681:19)
	at require (internal/modules/cjs/helpers.js:16:16)
	at Function.s [as __$__nodeRequire] (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/loader.js:32:963)
	at d._loadCommonJSModule (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:808:289)
	at d._doActivateExtension (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:712:547)
	at d._activateExtension (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:711:582)
	at Object.actualActivateExtension (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:709:534)
	at d._activateExtension (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:535:21)
	at /home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:534:632
	at Array.map (<anonymous>)
	at d._activateExtensions (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:534:620)
	at d.activateByEvent (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:532:413)
	at d._activateByEvent (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:710:282)
	at d._handleEagerExtensions (/home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:714:122)
	at /home/tkadur/.vscode-server/bin/26076a4de974ead31f97692a0d32f90d735645c0/out/vs/server/remoteExtensionHostProcess.js:717:532
	at processTicksAndRejections (internal/process/task_queues.js:89:5)
```
</details>

**Comment by KolyaKorruptis** [comment]
Possibly a duplicate of #43

**Comment by spmeesseman** [comment]
it will happen when the tree is rebuilding, this has been fixed in 1.23.  ill keep #43 open for now, I have not played with remote setup yet
